docs/roadmap.md

```diff
@@ -9,33 +9,17 @@ items from this roadmap.
 Description|Status|Links|
 -----------|------|-----|
 **Testing**|
-Add metric testing to the testbed|Done|
-Implement matrix testing in the testbed and convert benchmarks for receivers and exporters to use it|Done|
-Implement E2E tests for all core formats (Jaeger, Zipkin, OpenCensus, OTLP, Prometheus)| |[#482](https://github.com/open-telemetry/opentelemetry-collector/issues/482)
-Improve comparison with baseline performance and fail CI build on degradations| |
-Add code coverage collection to Contrib repository| |
-Add 2-stage build pipeline: separate build and certification jobs| |
-| |
-**Performance**|
-Convert internal memory representation to OTLP Protobuf|In progress |[#478](https://github.com/open-telemetry/opentelemetry-collector/issues/478)
-Research: explore Fast pass-through mode| |[Partial Protobufs](https://blog.najaryan.net/posts/partial-protobuf-encoding/)
-Research: alternative compression algorithms| |
-| |
-**Observability**|
-Add pipeline metrics| |[observability.md](observability.md)
-High-level observations|Draft|[#485](https://github.com/open-telemetry/opentelemetry-collector/issues/485)
-Research: Explore ways to expose pipelines state as a graph| |[#486](https://github.com/open-telemetry/opentelemetry-collector/issues/486)
+Metrics correctness tests|In progress|[#652](https://github.com/open-telemetry/opentelemetry-collector/issues/652)
 | |
 **New Formats**|
-OTLP Receiver and Exporter| |[#480](https://github.com/open-telemetry/opentelemetry-collector/issues/480) [#479](https://github.com/open-telemetry/opentelemetry-collector/issues/479)
-Research: OTLP over HTTP 1.1| |
```

Comment:
This is missing the exporter

Reply:
The "Complete OTLP support" line includes that.

Reply:
Just to make it clear: it was a research, now the research is done. Now it is part of OTLP spec. When we say "OTLP support" it means OTLP/HTTP must be also supported.

Reply:
If the text is unclear I can try to clarify.

Reply:
Clarified.